File: February/24-02-2024/main.py
print("1 --> SUM\n2 --> SUB\n3 --> MUL\n4 --> DIV\n5 --> REM\n6 --> END")
option = 0

while option != 6:

    option = int(input("Enter option : "))

    if option == 6:
        print("Bye.\n")
        break

    num1 = int(input("Enter num1 : "))
    num2 = int(input("Enter num2 : "))

    if option == 1:
        print(f"{num1} + {num2} = {num1 + num2}\n")

    elif option == 2:
        print(f"{num1} - {num2} = {num1 - num2}\n")
    
    elif option == 3:
        print(f"{num1} * {num2} = {num1 * num2}\n")
    
    elif option == 4:
        print(f"{num1} / {num2} = {num1 / num2}\n")
    
    elif option == 5:
        print(f"{num1} % {num2} = {num1 % num2}\n")

File: February/24-02-2024/main.js
import readline from "readline-sync";

console.log("1 --> SUM\n2 --> SUB\n3 --> MUL\n4 --> DIV\n5 --> REM\n6 --> END");
var option = 0;
var num1, num2;

while (option != 6) {

    option = parseInt(readline.question("Enter option : "));

    if (option == 6) {
        console.log("Bye.\n");
        break;
    }

    num1 = parseInt(readline.question("Enter num1 : "));
    num2 = parseInt(readline.question("Enter num2 : "));

    switch (option) {
        case 1:
            console.log(`${num1} + ${num2} = ${num1 + num2}\n`);
            break;

        case 2:
            console.log(`${num1} - ${num2} = ${num1 - num2}\n`);
            break;

        case 3:
            console.log(`${num1} * ${num2} = ${num1 * num2}\n`);
            break;

        case 4:
            console.log(`${num1} / ${num2} = ${num1 / num2}\n`);
            break;
        
        case 5:
            console.log(`${num1} % ${num2} = ${num1 % num2}\n`)
            break;

        default:
            console.log("Invalid operation.");

    }




}
